msteams-platform/task-modules-and-cards/cards/people-picker-adaptive-card.md

```diff
@@ -0,0 +1,246 @@
+---
+title: People Picker in Adaptive Cards
+description: Describes how to use the People Picker control in Adaptive Cards
+localization_priority: Normal
+keywords: Adaptive Cards People Picker
+ms.topic: reference
+author: Rajeshwari-v
+ms.author: surbhigupta
+---
+
+# People Picker in Adaptive Cards
+
+> [!NOTE]
+> This feature is available in [public developer preview](~/resources/dev-preview/developer-preview-intro.md) only.
+
+People Picker helps users to search and select users in Adaptive Card. You can add People Picker as input control to Adaptive Card, which works across chats, channels, task modules, and tabs. People Picker supports the following features:        
+
+* Searches single or multiple users.
+* Selects single or multiple users. 
+* Reassigns to single or multiple users. 
+* Prepopulates the name of selected users.
+
+The following image demonstrates selection of multiple users and the response Adaptive Card in People Picker with **Conversation Member Search**:
+
+![People Picker conversation member search](../../assets/images/cards/peoplepicker-conversation-member-search.png)
```

Comment:
@shiladityasaha can you help with a scenario based screenshot here to illustrate people picker. Since this is in the intro to the feature, the image needs to be descriptive.

Reply:
Done